README.md

```diff
@@ -101,6 +101,10 @@ pod/es-ingest-548b65475-whvqx   1/1       Running   0          7m
 pod/es-master-879576496-dhnlp   1/1       Running   0          7m
 pod/es-master-879576496-jjlvf   1/1       Running   0          7m
 pod/es-master-879576496-sgwxf   1/1       Running   0          7m
+
+kubectl get configmaps
+NAME                DATA      AGE
+kibana-config       1         10m
```

Comment:
Kibana is optional, so this shouldn't be there as it could confuse others. Please, remove.